examples/config.yaml

```diff
@@ -1,8 +1,5 @@
 config:
-  clusterimageset: openshift-v4.9.0
-  OC_OCP_VERSION: "4.9"
-  OC_OCP_TAG: "4.9.0-x86_64"
-  OC_RHCOS_RELEASE: "49.84.202110081407-0" # TODO automate it to get it automated using binary
+  OC_OCP_VERSION: "4.10.6"
   OC_ACM_VERSION: "2.4"
   OC_OCS_VERSION: "4.9"
```

Comment:
I would put some indicator here to remark that you cannot use (for now) other version of OCS